src/complex/Common/TypesUtility.hpp

```diff
@@ -247,6 +248,72 @@ inline constexpr StringLiteral DataTypeToString(DataType dataType)
   }
 }
 
+inline const std::list<std::string>& GetAllDataTypesAsStrings()
```

Comment:
Why is this a list instead of a vector? This takes advantage of none of the benefits of a std::list but gets none of the advantages of continuous memory like what std::vector provides.